rero_ils/modules/items/api/api.py

```diff
@@ -157,25 +160,40 @@ def get_item_record_for_ui(cls, **kwargs):
         return item
 
     @classmethod
-    def get_items_with_obsolete_temporary_item_type(cls, end_date=None):
-        """Get all items with an obsolete temporary item_type.
+    def format_end_date(cls, end_date):
+        """Return a formatted end date."""
+        if end_date is None:
+            end_date = datetime.utcnow()
```

Comment:
according @jma, it's better to use `datetime.now(timezone.utc)` instead of `datetime.utcnow()`